Directions.md

```diff
@@ -33,3 +33,56 @@ better. Also make sure to slide the wings on the skewer as they tend to stick to
 9) Once everything is cooked, you can put the skewers between the slices of bread and pull the skewer out. This will leave the wings between the slices of bread and they are ready to be served. 
```

Comment:
I'd like to see more information about the Grill here.  @derekquinn can you make that an action item?  I won't hold up this PR, since it's out of scope.